src/com/lftechnology/java/training/krishna/employeemanagement/controllers/LoginController.java

```diff
@@ -0,0 +1,474 @@
+
+package com.lftechnology.java.training.krishna.employeemanagement.controllers;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+import com.lftechnology.java.training.krishna.employeemanagement.domain.Employee;
+import com.lftechnology.java.training.krishna.employeemanagement.domain.User;
+import com.lftechnology.java.training.krishna.employeemanagement.service.employee.impl.EmployeeServiceImpl;
+import com.lftechnology.java.training.krishna.employeemanagement.service.user.impl.UserServiceImpl;
+import com.lftechnology.java.training.krishna.employeemanagement.utils.ConstantUtils;
+import com.lftechnology.java.training.krishna.employeemanagement.utils.Role;
+import com.lftechnology.java.training.krishna.employeemanagement.utils.ValidationUtils;
+
+/**
+ *  The application should have the following features:
+ * 1. Application will have two users: a. admin user b. normal user
+ * 2. Admin user will have option to add/delete user
+ * 3. Normal user can view/search employee
+ * 4. admin user will be generated by the system by default
+ * 5. admin user will add normal user
+ */
+
+/**
+ * This program used for simple employeement system
+ *
+ * @author Krishna Timilsina <krishnatimilsina@lftechnology.com>
+ * @version 1.0
+ */
+public class LoginController {
+
+	private static UserServiceImpl userServiceImpl;
+	private static EmployeeServiceImpl employeeServiceImpl;
+	private static User user;
+
+	
+	//constructor
+	private LoginController() {
+
+	
+	}
+
+	public static void main(String[] args) {
+
+		Scanner scanner = new Scanner(System.in);
+		authentiateUser(scanner);
+		scanner.close();
+
+	}
+
+	/**
+	 * This method is used to authenticate user
+	 *
+	 * @param scanner
+	 *            {@link Scanner}
+	 * @author Krishna Timilsina <krishnatimilsina@lftechnology.com>
+	 */
+	private static void authentiateUser(Scanner scanner) {
+
+		showLoginTitle();
+		user = loginAction(scanner);
+		while (user == null) {
+			System.out.println("Invalid username and password. Please try again.\n");
+			loginAction(scanner);
+		}
+		showMainTitle(user.getUsername());
+		showMainMenu(scanner, user.getId());
+	}
+
+	/**
+	 * This method is used to display title of login
+	 *
+	 * @author Krishna Timilsina <krishnatimilsina@lftechnology.com>
+	 */
+	private static void showLoginTitle() {
+
+		System.out.println("\n"
+			+ "****************************************************" + "\n"
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "****************************************************" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)